linera-service/src/cli_wrappers.rs

```diff
@@ -742,25 +739,46 @@ impl LocalNetwork {
     }
 
     async fn run_server(&mut self, i: usize, j: usize) -> Result<Child> {
-        let storage = match self.database {
-            Database::RocksDb => format!("rocksdb:server_{}_{}.db", i, j),
-            Database::DynamoDb => format!("dynamodb:server_{}_{}.db:localstack", i, j),
-            Database::ScyllaDb => format!("scylladb:table_server_{}_{}_db", i, j),
+        let (storage, key) = match self.database {
+            Database::RocksDb => (format!("rocksdb:server_{}_{}.db", i, j), (i, j)),
+            Database::DynamoDb => (
+                format!("dynamodb:{}_server_{}.db:localstack", self.table_name, i),
+                (i, 0),
+            ),
+            Database::ScyllaDb => (
+                format!("scylladb:{}_server_{}_db", self.table_name, i),
+                (i, 0),
+            ),
         };
-        let key = (i, j);
         if !self.set_init.contains(&key) {
```

Comment:
So with DynamoDB and ScyllaDB only the first shard will get created? Is there a plan to make this multi-shard again in the future?